src/fragments/lib/auth/android/signin_next_steps/60_confirm_signup.mdx

```diff
@@ -0,0 +1,27 @@
+If you receive `CONFIRM_SIGN_UP` as a next step, sign up could not proceed without confirming user information such as email or phone number. The next step is to invoke the `confirmSignUp` API and follow the confirm signup flow.
+
+<BlockSwitcher>
+
+<Block name="Kotlin">
+
+```kotlin
+ try {
+      Amplify.Auth.confirmSignUp(
+          "username",
+          "confirmation code",
+          { result ->
+              Log.i(
+                  TAG,
+                  "Confirm signUp result completed: ${result.isSignUpComplete}"
+              )
```

Comment:
nit: Are we crossing the line limit here? if yes, what is the limit?

Reply:
There is no limit in the docs. I will remove the line breaks.

Reply:
done